cypress/integration/analytics-tables/index.js

```diff
@@ -0,0 +1,50 @@
+import { Before, Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const JOB_ID = 'JOB_ID'
+
+Before(() => {
+    cy.clock()
+
+    cy.intercept(
+        { pathname: /resourceTables\/analytics$/, method: 'POST' },
+        req => {
+            req.reply(200, {
+                response: {
+                    id: JOB_ID,
+                },
+            })
+        }
+    ).as('startAnalyticsTablesExportTask')
+
+    cy.intercept(
+        { pathname: /system\/tasks\/ANALYTICS_TABLE/, method: 'GET' },
+        req => {
+            expect(req.url.endsWith(JOB_ID)).to.be.true
+            req.reply(200, [
+                {
+                    id: JOB_ID,
+                    level: 'INFO',
+                    message: 'Analytics tables updated',
+                    completed: true,
+                },
+            ])
+        }
+    ).as('pollAnalyticsTablesExport')
+})
+
+Given('the user navigated to the analytics tables page', () => {
+    cy.visit('/#/analytics')
+    cy.findByRole('heading', { name: 'Analytics tables management' }).should(
+        'exist'
+    )
+})
+
+When(`the user clicks on the 'Start export' button`, () => {
+    cy.findByRole('button', { name: 'Start export' }).click()
+})
+
+Then('the tables are exported', () => {
+    cy.wait('@startAnalyticsTablesExportTask')
+    cy.tick(5000)
```

Comment:
I assume that this is because of the `PULL_INTERVAL` constant? I can imagine that either an explanatory comment, or importing and using the PULL_INTERVAL here would make it clearer what's going on. I think I'd probably lean towards the comment.

Reply:
Or, if there'd be a way to not have to deal with timeouts at all, but instead wait for some content to appear (or something like that), that seems even better (to me).

Reply:
That's right - this is because the first poll occurs after `PULL_INTERVAL` milliseconds. The export button does change its text from 'Start export' to 'Exporting...' once the user clicks on it, which we could use instead of waiting for the poll to occur, but I'd like to verify that a network request to the right endpoint occurs so I'll go with the comment.

Reply:
Sounds good 👍